### PR TITLE
Some cleaning

### PR DESCRIPTION
**Pull request by arjantijms**
* Order pom.xml
* Remove unnecessary profile in pom.xml
* Make checkstyle.xml same as in GlassFish
* Some formatting and mild cleaning in utils classes en exec mojo

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -81,31 +121,12 @@
...
-                        </dependency>
-                    </dependencies>
-                </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>3.5.0</version>
                     <configuration>
-                        <additionalOptions>${javadoc.options}</additionalOptions>
+                        <additionalOptions>-Xdoclint:none</additionalOptions>
```
Comment (senivam) [inline pom.xml:129]: should be ``<doclint>none</doclint>``

File: src/main/java/org/glassfish/build/utils/ZipHelper.java [hunk 2]
```diff
@@ -46,7 +47,7 @@ private ZipHelper() {
     /**
      * Lazy singleton holder.
      */
-    private static class LazyHolder {
+    private static final class LazyHolder {
```
Comment (senivam) [inline src/main/java/org/glassfish/build/utils/ZipHelper.java:50]: final is useless here

**Review by senivam — COMMENTED** [review]
In general looks good, but I have 2 notes. Would appreciate it if they were addressed. :)

**Comment by dmatej** [comment]
Replaced by #145

**Pull request closed without merging**